### ISSUE TITLE
Formatting issue with Lex.

### ISSUE THREAD

**Issue by iamurthy** [aws-samples/aws-lex-web-ui#690]
We are trying to show the response from lambda function hooked up to aws lex bot, but we are running into formatting issue. We have string that is sent from lambda which as \n\n for new lines at 2 different places. The first \n\n is recognized and a new lines is printed in the frontend but the \n\n followed by bullet points is not recognized.

Structure of the content sent from lambda function is 
paragraph\n\n -- New line is printed here
bullet points\n\n
paragraph -- New line is not printed before this paragraph.

**Comment by atjohns** [comment]
Do you have an example of what your custom payload looks like. I just tried to test and it was displaying correctly for me...

Payload
`messageArray = [{'contentType': 'CustomPayload', 'content': 'paragraph\n\nbullet points\n\nparagraph'}]
`

Response in Web UI
![image](https://github.com/aws-samples/aws-lex-web-ui/assets/94479740/7c162c38-6e95-4432-9697-0655bade6aed)

**Comment by iamurthy** [comment]
Sample response from the lambda function is below.

{'sessionState': {'dialogAction': {'type': 'ElicitIntent'}, 'intent': {'name': 'Chat', 'state': 'Fulfilled'}}, 'messages': [{'contentType': 'CustomPayload', 'content': "To validate your membership, I will need you to provide some personal details so I can look up your account information in our system. Specifically, I'll need your:\n\n- First name\n- Last name\n- Date of birth (in the format YYYY-MM-DD)\n- Last 4 digits of your social security number\n\nOnce you've provided that information, I can look up your membership details and validate your account. Please go ahead and provide those details whenever you're ready."}]}

**Comment by atjohns** [comment]
Yes looks like this specific text does not render properly. Still investigating root cause as the HTML looks correct that is being rendered:

`"<p>To validate your membership, I will need you to provide some personal details so I can look up your account information in our system. Specifically, I&#39;ll need your:</p>
<ul>
<li>First name</li>
<li>Last name</li>
<li>Date of birth (in the format YYYY-MM-DD)</li>
<li>Last 4 digits of your social security number</li>
</ul>
<p>Once you&#39;ve provided that information, I can look up your membership details and validate your account. Please go ahead and provide those details whenever you&#39;re ready.</p>
"`

**Comment by atjohns** [comment]
Ok figured it out, its a CSS issue. I will get a fix out in the next release but if you need it sooner then you can modify your `custom-chatbot-style.css` in your S3 bucket and add the following line

```
.message-text :deep(p) {
  margin-top: 16px !important;
}
```

**Comment by iamurthy** [comment]
I added the above code in custom-chatbot-style.css, now my file looks like below.

button.min-button {border-radius: 60px;}

.message-button {display: none;}

.message-text :deep(p) {
  margin-top: 16px !important;
}

Can you please check above and let know if the config looks good.
I still see the same issue, Do we need to re-deploy the stack for the changes to be reflected?

![image](https://github.com/aws-samples/aws-lex-web-ui/assets/40452370/d39212ff-ebfe-4622-97f0-abaaa2c29f1c)

**Comment by atjohns** [comment]
Apologies you don't need the deep selector just do...

`.message-text p {
margin-top: 16px !important;
}`

**Comment by iamurthy** [comment]
@atjohns Thank you, it worked now.
